### DISCUSSION TITLE
Crowdin proofreader for Filipino `fil`

### DISCUSSION THREAD

**Discussion by IverCoder** [d4rken-org/sdmaid-se#162] [General]
Hello, can I be a Crowdin proofreader for Filipino (`fil`)? I have translated some strings there and want to ensure that the translations don't get overwritten by somebody else writing less appropriate translations. My username on Crowdin is [@IverCoder](https://crowdin.com/profile/IverCoder).

**Comment by d4rken** [comment]
Done, thanks for your help :heart:

**Reply by IverCoder** [reply]
Thank you too for open-sourcing this app! 💗
